frontend/src/composables/useShootEditor/helper.js

```diff
@@ -83,9 +90,9 @@ export class EditorCompletions {
         end,
         string,
       }
-      if (token.start <= pos.ch && pos.ch <= token.end) {
+      if (token.start <= lineChar && lineChar <= token.end) {
         // Ensure that mouse pointer is on propety and not somewhere else on this line
-        const completions = this._getYamlCompletions(token, pos, cm, true)
+        const completions = this._getYamlCompletions(token, { ch: lineChar, line }, cmView, true)
```

Comment:
extract to variable for better readability
```suggestion
        const cur = { ch: lineChar, line }
        const completions = this._getYamlCompletions(token, cur, cmView, true)
```

Reply:
What is cur? Is it current? Or is it cursor? Use easy understable names.

Reply:
I think this is a leftover convention from cm5 where they used `cur` for cursor in some places. Anyway, I changed all occurrences of `cur` to `cursor` in our code.